js/pages/configuration/configuration.js

```diff
@@ -113,11 +118,25 @@ define([
 
     async onPageCreated() {
       this.loading(true);
+      const info = await buildInfoService.getBuildInfo();
       await sourceApi.initSourcesConfig();
       super.onPageCreated();
+      this.isSolrVocabEnabled = info.configuration.vocabulary.solrEnabled && info.configuration.vocabulary.hasOwnProperty("cores");
+      this.isSolrVocabCoreAvailable = false;
+      if (this.isSolrVocabEnabled) {
+        this.solrVocabCoreName = this.getSolrVocabCoreName(info);
+      }
       this.loading(false);
     }
 
+    getSolrVocabCoreName(info) {
+      if (typeof(info.configuration.vocabulary.cores) != "string") {
+        this.isSolrVocabCoreAvailable = true;
```

Comment:
I think i found where isSolrVocabCoreAvailable is assigned to true here.  Note, it's not observable, you're just storing a boolean value.   This may work if this all happens when the page loads (ie: the configuration page loads) and there is no change in the isSolrVocabCoreAvailable state while the page is alive.  If that's the case, don't make anything observable.